bootstrap/017_query_history.sql

```diff
@@ -166,3 +170,44 @@ BEGIN
         call internal.set_config(:key, 'true');
     end if;
 END;
+
+create or replace procedure internal.update_qtag_day()
+returns number
+language sql
+comment = 'materialize qtag_filter for a single day until fully backfilled'
+as
+begin
+UPDATE internal_reporting_mv.query_history_complete_and_daily
+SET    qtag_filter=tools.qtag_to_map(qtag)
+WHERE  start_time::date =
+       (
+              SELECT max(start_time::date)
+              FROM   internal_reporting_mv.query_history_complete_and_daily
+              WHERE  qtag_filter IS NULL
+              AND    qtag IS NOT NULL)
+AND    qtag_filter IS NULL
+AND    qtag IS NOT NULL;
+
+let updates number :=
+(
+       SELECT $1
+       FROM   TABLE(result_scan(last_query_id())));
+
+
+UPDATE internal_reporting_mv.query_history_complete_and_daily_incomplete
```

Comment:
i don't think we should update incomplete. It gets dropped and recreated every hour so no point.